wip_pacta_data_qa.Rmd

```diff
@@ -0,0 +1,545 @@
+datatable(no_ALD_EQ)
+
+no_ALD_EQ_detail <- result_data_qa %>% 
+  filter(asset_type == "Equity" &
+           has_asset_level_data == F &
+           financial_sector != "Other" &
+           investor_name != "Meta Investor") %>% 
+  select(financial_sector, isin, company_name, market_value, value_usd) %>% 
+  group_by(financial_sector, isin, company_name) %>% 
+  summarise(sum_value_usd = round(sum(value_usd, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(share_total_value = round(
+    sum_value_usd/(total_overview %>% filter(asset_type=="Equity") %>% pull(sum_value_usd)),6)) %>% 
+  arrange(financial_sector, desc(share_total_value))
+datatable(no_ALD_EQ_detail)
+
+
+```
+
+### overview Corporate Bond holdings w/o ALD
+Overview of Holdings in PACTA sectors for which no ALD could be matched. Removing ISIN here, one company can have multiple ISINs on bond level
+
+1. Overview on sector level, including count and share of overall corporate bonds value affected
+2. Overview on sector-company level, including overall corporate bonds value affected
+
+**Actions:**
+
+* if there are companies present, for which we should have corresponding asset-level data, check in DataStore if the source of the issue can be located. Possibly involve Taylor to fix such mapping
+* other steps???
+
+```{r}
+no_ALD_CB <- result_data_qa %>% 
+  filter(asset_type == "Bonds" &
+           has_asset_level_data == F &
+           financial_sector != "Other" &
+           investor_name != "Meta Investor") %>% 
+  group_by(financial_sector) %>% 
+  summarise(total_no_ALD_CB = n(), sum_value_usd = round(sum(value_usd, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(share_total_value = round(
+    sum_value_usd/(total_overview %>% filter(asset_type=="Bonds") %>% pull(sum_value_usd)),6)) %>% 
+  arrange(financial_sector, desc(share_total_value))
+datatable(no_ALD_CB)
+
+no_ALD_CB_detail <- result_data_qa %>% 
+  filter(asset_type == "Bonds" &
+           has_asset_level_data == F &
+           financial_sector != "Other" &
+           investor_name != "Meta Investor") %>% 
+  #select(financial_sector, isin, company_name, market_value, value_usd) %>% 
+  select(financial_sector, company_name, market_value, value_usd) %>% 
+  #group_by(financial_sector, isin, company_name) %>% 
+  group_by(financial_sector, company_name) %>% 
+  summarise(sum_value_usd = round(sum(value_usd, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(share_total_value = round(
+    sum_value_usd/(total_overview %>% filter(asset_type=="Bonds") %>% pull(sum_value_usd)),6)) %>% 
+  arrange(financial_sector, desc(share_total_value))
+datatable(no_ALD_CB_detail)
+
+
+```
+
+## Non-PACTA sector with ALD
+
+### overview Equity holdings, non-PACTA sector but with ALD
+Overview of Holdings in non-PACTA sectors for which we do find ALD.
+
+1. Overview on sector level, including count and share of overall equity value affected
+2. Overview on sector-company level, including overall equity value affected
+
```

Comment:
I think this also requires an additional sentence or two about what this is/means